Update information (#1)
OpenScienceLabs/gsoc: organization-questionnaire/gsoc2025_proposal.md

teaching, learning, and sharing information about open science and computational
tools. Our community develops tools that address real-world problems and
collaborates with other projects and workgroups to improve technology and create
international opportunities for our community. Although our focus may seem
broad, we initially prioritize supporting Research Software Engineers (RSEs) who
often face computational challenges in their work. We recognize that many
colleagues in scientific fields may not be familiar with programming languages,
computational libraries, version control systems, databases, DevOps, and other
computational tools. Therefore, we aim to provide a safe and open space for
individuals to learn and share their knowledge. Our community is open to
everyone, including students, professors, and industry professionals, as we
believe that the challenges and technologies used in these fields are frequently
similar, if not the same (in many cases).

## Primary open source license

BSD 3 Clause: <https://github.com/osl-incubator/scicookie/blob/main/LICENSE>

## Why does your org want to participate in GSoC?

> Open Science Labs aims to participate in Google Summer of Code (GSoC) to further
> its mission of promoting open science and collaborative development. By
> participating in the program, the organization is able to attract students from
> diverse academic backgrounds and skill levels, enabling them to contribute their
> unique perspectives and skills to open source projects. This not only enriches
> the open source ecosystem, but also allows students to apply theoretical
> knowledge to practical, high-impact projects, benefiting the global science and
> technology community. Additionally, Open Science Labs will be joining forces
> with open science and open-source projects that have great potential but often
> lack the necessary visibility.
> 
> GSoC provides an ideal platform for large and small projects to showcase and
> promote their unique functionalities. As an organization committed to open
> science and open source, one of its main goals is to create and improve tools
> that are accessible to everyone. Through GSoC, students can work on projects
> ranging from improving existing tools to developing new features, directly
> contributing to the advancement of open science and reaching a global audience.
> 
> Mentorship is another key aspect of Open Science Labs involvement in GSoC. By
> providing guidance and support, the organization helps students to develop
> professionally while introducing them to the world of open source software
> development. This mentorship reinforces Open Science Labs’ commitment to
> education and talent development, ensuring that students gain valuable skills
> and a deep understanding of how to effectively contribute to open source
> projects.
> 
> In addition, GSoC fosters a sense of community and collaboration that aligns
> with Open Science Labs' values. Participation in the program allows the
> organization to expand its network of collaborators and cultivate an environment
> of teamwork that extends beyond the duration of the program. This
> community-driven approach strengthens connections between participants and
> fosters a culture of ongoing collaboration in science and technology.
> 
> In summary, GSoC enables Open Science Labs to innovate and develop through
> proposed initiatives, also to sell open technological know-how, guide open
> supply software program improvement, guide students of their expert growth,
> strengthen worldwide collaboration, and foster innovation in technological
> know-how and technology.

Author: ```suggestion
Open Science Labs aims to participate in Google Summer of Code (GSoC) to further its mission of promoting open science and collaborative development. By participating in the program, the organization is able to attract students from diverse academic backgrounds and skill levels, allowing them to contribute their unique perspectives and skills to open source projects. This not only enriches the open source ecosystem, but also allows students to apply theoretical knowledge to practical, high-impact projects, benefiting the global science and technology community. In addition, Open Science Labs will join forces with open science and open source projects that have great potential but often lack the necessary visibility, acting as an umbrella organization.
```

broad, we initially prioritize supporting Research Software Engineers (RSEs) who
often face computational challenges in their work. We recognize that many
colleagues in scientific fields may not be familiar with programming languages,
computational libraries, version control systems, databases, DevOps, and other
computational tools. Therefore, we aim to provide a safe and open space for
individuals to learn and share their knowledge. Our community is open to
everyone, including students, professors, and industry professionals, as we
believe that the challenges and technologies used in these fields are frequently
similar, if not the same (in many cases).

## Primary open source license

BSD 3 Clause: <https://github.com/osl-incubator/scicookie/blob/main/LICENSE>

## Why does your org want to participate in GSoC?

Open Science Labs aims to participate in Google Summer of Code (GSoC) to further
its mission of promoting open science and collaborative development. By
participating in the program, the organization is able to attract students from
diverse academic backgrounds and skill levels, enabling them to contribute their

> unique perspectives and skills to open science projects. This not only enriches

Reviewer: ```suggestion
unique perspectives and skills to open source projects. This not only enriches
```

> the Open Science Labs ecosystem, but also allows students to apply theoretical

Reviewer: ```suggestion
the open source ecosystem, but also allows students to apply theoretical
```

knowledge to practical, high-impact projects, benefiting the global science and
technology community. Additionally, Open Science Labs will be joining forces
with open science and open-source projects that have great potential but often
lack the necessary visibility.

> GSoC provides an ideal platform for Open Science Labs and small projects to

Reviewer: ```suggestion
GSoC provides an ideal platform for large and small projects to
```

> showcase and promote open source software development projects. As an

Reviewer: ```suggestion
showcase and promote their unique functionalities. As an
```

> organization committed to open science, one of its main goals is to create and

Reviewer: ```suggestion
organization committed to open science and open source, one of its main goals is to create and
```

fosters a culture of ongoing collaboration in science and technology.

In summary, GSoC enables Open Science Labs to innovate and develop through
proposed initiatives, also to sell open technological know-how, guide open
supply software program improvement, guide students of their expert growth,
strengthen worldwide collaboration, and foster innovation in technological
know-how and technology.

## What would your organization consider to be a successful GSoC for your org?

Open Science Labs would consider GSoC successful if it achieves meaningful
outcomes in line with its mission. This includes empowering students by
providing them with mentorship, guidance and the opportunity to develop valuable
skills in open source development. Success would also be reflected in the
advancement of open source tools, with students contributing to the improvement
of existing tools or the creation of new ones that benefit the open science
community. In addition, fostering a sense of collaboration and community among
participants, both during and after the program, would be a key indicator of
success. Ultimately, a successful GSoC would not only drive innovation and

> growth for Open Science Labs, but also inspire students to continue contributing

Reviewer: ```suggestion
growth for the open source projects, but also inspire students to continue contributing
```

> to open science and open source projects, further increasing the organization's

Reviewer: ```suggestion
to these projects, further increasing the organization's
```

gain fresh perspectives from students. By aligning mentorship with our formula
for success - empowering the community through tools, resources and support - we
ensure that mentors stay motivated and invested in helping contributors succeed,
ultimately contributing to the lasting impact of open science and technology.

## How will you help your GSoC contributors stay on schedule to complete their projects?

To help GSoC contributors stay on schedule and complete their projects
successfully, Open Science Labs would implement a structured yet flexible
approach. This would include establishing clear milestones and deliverables at
the outset, scheduling regular check-ins for progress reviews and feedback, and
encouraging the use of project management tools. Mentors would actively guide
contributors, fostering open communication and providing support to overcome
challenges. By breaking down larger goals into manageable tasks and maintaining
a supportive environment, Open Science Labs would ensure that contributors stay
motivated, productive, and on track to deliver impactful results. Also, our
organization has members from multiple countries, and most meetings are held in

> Spanish, Portuguese and English to accommodate everyone. We will invite every

Reviewer: maybe not necessary to mention about these other languages, because the official language in the event is English

selected student to these meetings and make them feel welcome.

## How will you keep your GSoC contributors involved in your community during and after GSoC?

To keep GSoC contributors involved in the Open Science Labs community during and
after GSoC, we will leverage our welcoming and inclusive environment. Our
community includes members from multiple countries, and meetings are held in

> Spanish, English, and Portuguese to accommodate everyone. We will invite each

Reviewer: not necessary to mention about language here .. or maybe mention that in a "after gsoc contribution" section .. I mean .. maybe at the end or in the part we mention about after gsoc contributions ... we can mention that osl members are encouraged to organize meetings and activities in their own language to foster the usage and development of the project in their own regions

selected student to these meetings to foster a sense of belonging. Regular
check-ins will be scheduled to discuss progress and provide feedback,
encouraging students to present their work in blogs and topical meetings. This
will help them gain confidence and interact with other contributors. We will
also share their work with our partners and on social media to boost visibility
within the open-source scientific community.

After GSoC, we will continue to provide mentorship and support, offering

> internships and inviting contributors to join study groups and new projects.

Reviewer: ```suggestion
internships, small grants, and inviting contributors to join study groups and new projects.
```

Community events, workshops, and webinars will keep contributors engaged, and an
alumni network will help them stay connected. By implementing these strategies,
Open Science Labs aims to create a supportive environment that keeps GSoC
contributors involved both during and after the program. Has your org been

> accepted as a mentor org in GSoC? Yes, Open Science Labs has been accepted as a

Reviewer: ```suggestion
accepted as a mentor org in GSoC? No
```

Actually no, we was accepted as a sub-org under NumFOCUS umbrella
